### PR TITLE
Provide support for Spring Boot configurations

### PR DESCRIPTION
**Pull request by aureamunoz**

### CODE REVIEW

File: docs/docs/microprofile-config.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Stork MicroProfile Config 
+
+Stork integrates with MicroProfile Configuration out of the box, enabling seamless access to configuration properties. 
+This documentation explains how Stork can retrieve configuration details from the MicroProfile Config file present in the classpath.
+
```
Comment (cescoffier) [inline docs/docs/microprofile-config.md:5]: Can you add something stating that this is the approached used in Quarkus?

File: docs/pom.xml [hunk 2]
```diff
@@ -78,6 +78,11 @@
             <groupId>io.smallrye.stork</groupId>
             <artifactId>stork-service-discovery-static-list</artifactId>
         </dependency>
+        <dependency>
+            <groupId>io.smallrye.stork</groupId>
+            <artifactId>stork-spring-boot-config</artifactId>
+<!--            <scope>provided</scope>-->
```
Comment (cescoffier) [inline docs/pom.xml:84]: Why is this commented out?

**Review by cescoffier — CHANGES_REQUESTED** [review]
Looks great! Just made two small comments.

File: docs/pom.xml (hunk 2 above)
Comment (aureamunoz) [reply docs/pom.xml]: Just testing something and forgot to un-comment then

**Review by cescoffier — APPROVED** [review]

**Comment by cescoffier** [comment]
Can you squash before merging?

**Comment by aureamunoz** [comment]
> Can you squash before merging?

ups, yes! done

**Pull request merged**